### PR TITLE
Fixed typo's, colons, unnecessary spaces and sort some lines

### PR DESCRIPTION
**Pull request by RonWillemse**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -84,54 +84,54 @@ html{font-family:sans-serif;-ms-text-size-adjust:100%;-webkit-text-size-adjust:1
 --------------------------------------------- */
 
 html,
-input[type="search"]{
+input[type="search"] {
 	-webkit-box-sizing: border-box;
```
Comment (GaryJones) [inline style.css]: These prefixes can be dropped anyway.

File: style.css [hunk 2]
```diff
@@ -174,11 +174,11 @@ input[type="submit"],
 textarea:focus,
 .button,
 .gallery img {
-	-webkit-transition: all 0.1s ease-in-out;
-	-moz-transition:    all 0.1s ease-in-out;
-	-ms-transition:     all 0.1s ease-in-out;
-	-o-transition:      all 0.1s ease-in-out;
-	transition:         all 0.1s ease-in-out;
+	-webkit-transition: all .1s ease-in-out;
```
Comment (GaryJones) [inline style.css]: > Use a leading zero for decimal values, including in rgba().
https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#values

File: style.css [hunk 3]
```diff
@@ -533,34 +533,34 @@ th:first-child {
...
-	z-index: 100000; /* Above WP toolbar. */
 	text-decoration: none;
-	box-shadow: 0 0 2px 2px rgba(0,0,0,.6);
+	width: auto;
+	z-index: 100000; /* Above WP toolbar. */
 }
 
 .more-link {
-    position: relative;
+		position: relative;
```
Comment (GaryJones) [inline style.css]: Two tabs here?

File: style.css [hunk 4]
```diff
@@ -84,54 +84,54 @@ html{font-family:sans-serif;-ms-text-size-adjust:100%;-webkit-text-size-adjust:1
...
-	-moz-box-sizing:    border-box;
-	box-sizing:         border-box;
+	-moz-box-sizing: border-box;
+	box-sizing: border-box;
 }
 
 *,
-*:before,
-*:after {
+*::before,
```
Comment (GaryJones) [inline style.css]: For those reading, this will drop IE8 support (yay!).

File: style.css (hunk 1 above)
Comment (RonWillemse) [inline style.css]: It is not my decision to remove lines from the style script, but you are right

File: style.css (hunk 3 above)
Comment (RonWillemse) [inline style.css]: My mistake, will fix that in the next commit.

File: style.css (hunk 4 above)
Comment (RonWillemse) [inline style.css]: First of all I hate IE8, second I like consistency and sometimes the script uses double colons and sometimes single. I prefer to use double and imo it is not possible to keep supporting very old browsers

File: style.css (hunk 2 above)
Comment (RonWillemse) [inline style.css]: Ok, it is in the WP standards, but personally, I chose to omit the zeros to make my CSS as clean as possible.

Comment (GaryJones) [inline style.css]: That's fine, but this codebase isn't only for your personal use ;-)

File: style.css (hunk 4 above)
Comment (GaryJones) [inline style.css]: You're right - there was already some `::before` in the file, so IE8 support looks to already have been dropped.

File: style.css (hunk 2 above)
Comment (RonWillemse) [inline style.css]: That's also fine, and I understand that the sample theme is a sort of allround starter theme, but it is a suggestion not a must.

Comment (GaryJones) [inline style.css]: And I'm giving a reason why this particular suggestion should be changed back to include a leading `0`, so there is more chance of the whole PR being accepted. There's no reason to deviate away from documented code standards for something this trivial.

Comment (RonWillemse) [inline style.css]: Ok, again I like consistency and I saw that there already were missing some leading zero's in the code before my PR. I will change it back to the WP Standards and re add a zero to the places where they already be missing too.

**Comment by GaryJones** [comment]
No explanation with the Close?

**Comment by bgardner** [comment]
@GaryJones Sorry, I was going off of https://github.com/copyblogger/genesis-sample/pull/60/commits/e8b68fa34dc1ff9e4b49a07801ac88628d8fb012 and missed other spacing issues were in previously.

Only thing I think is left is the conversation about : and :: right?

**Comment by GaryJones** [comment]
> Only thing I think is left is the conversation about : and :: right?

I've not checked to see which of the spacing / missing semi-colon / property order / leading zero / and so on have been fixed.

There shouldn't be much discussion needed for the colon. The spec says to use `::` for `before` and `after` (pseudo-elements) and `:` for pseudo-classes. The only reason to also use `:` for pseudo-elements was that the earlier spec used that, and so browsers supported that. Since some before/after were already using `::`, there's no point in keeping the `:` for the other before/after instances.

**Comment by RonWillemse** [comment]
This was all about consistency. At some lines there was : and at other lines there was ::

**Pull request closed without merging**